scripts/release-notes/release_notes.go

```diff
@@ -140,6 +140,8 @@ Download one of our static release bundles via our Google Cloud Bucket:
 		time.Now().Format(time.RFC1123),
 		bundleVersion, bundleVersion,
 		bundleVersion, bundleVersion,
+		bundleVersion, bundleVersion,
+		bundleVersion, bundleVersion,
```

Comment:
this would obselete https://github.com/cri-o/cri-o/pull/5629 which I'm fine with